package.json

```diff
@@ -23,7 +23,8 @@
 		"postpack": "npm run clean",
 		"build": "rspack build",
 		"typecheck": "tsc --noEmit",
-		"test": "node --test ./test"
+		"test": "node --test --experimental-test-snapshots ./test/index.test.js",
+		"test:regenerate": "node --experimental-test-snapshots --test-update-snapshots ./test/index.test.js"
```

Comment:
Same here

Reply:
```js
"test": "node --test ./test"
```

ı used this commant, but get a this error in tests

```js
ℹ tests 1
ℹ suites 0
ℹ pass 0
ℹ fail 1
ℹ cancelled 0
ℹ skipped 0
ℹ todo 0
ℹ duration_ms 86.965917

✖ failing tests:

test at test:1:1
✖ test (79.325333ms)
  'test failed'
➜  amaro git:(dev-1) ✗ 
```

Reply:
Thats odd try to remove the path completely

Reply:
```js
"test": "node --test --experimental-test-snapshots",
"test:regenerate": "node --test --experimental-test-snapshots --test-update-snapshots ./test"
```

unfortunately I failed I updated this way

Reply:
this problem was solved in your last suggestion thank you